viewing_party/party.py

```diff
@@ -1,23 +1,226 @@
 # ------------- WAVE 1 --------------------
+import json
+from unittest import result
 
 def create_movie(title, genre, rating):
-    pass
+    '''
+    Input: movie title, genre, and rating
+    Ouput: If values truthy, produces a dictionary of movies with title, genre, and rating key-value pairs.
+    If inputs are falsy, function returns None.
+    '''
+    movie_dict = {}
+    # if title == True and genre == True and rating == True:
+    if True: 

```

Comment:
Because `True` is hardcoded here, we will always enter the body of this conditional. This means it isn't necessary to even have the conditional statement since the code inside will always run.